runtimes/cloudformation/go.mod

```
@@ -42,3 +42,5 @@ require (
 	golang.org/x/sys v0.9.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/falcosecurity/kilt/pkg => ../../pkg
```

```suggestion
replace github.com/agent-kilt/kilt/pkg => ../../pkg
```

Or something similar?